authors/apps/profiles/models.py

```diff
@@ -30,6 +34,24 @@ class Profile(models.Model):
 
     def __str__(self):
         return str(self.user.username)
+    #follows a user
+    def follow(self, profile):
+        self.is_following.add(profile)
+
+    #unfollows a user
+    def unfollow(self, profile):
+        self.is_following.remove(profile)
+
+    #append the profile user list to get the user followers
+    def get_following(self, profile=None):
+        users = self.is_following.all()
+        profile_list = []
+        for item in users:
```

Comment:
@Deekerubo good implementation however this could be well-achieved using list comprehension so as to minimize code complexity

Reply:
Done